less/mixins.less

```diff
@@ -1 +1,16 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+
+//MINIMUM 2, 1 OF WHICH IS PARAMETRIC FOR THE BUTTON
+.flex(@direction){
+    display: flex;
+    flex-direction: @direction;
+}
+.sign-up(@width, @padding){
```

Comment:
The mix in might also be more reusable on a big project if you also passed in colors